### PR TITLE
change: adding eu-central-2 bucket info to JS constants

### PR DESCRIPTION
**Pull request by ErnevSharma**
*Issue #, if available:*

*Description of changes:*

Adding JS bucket info. I verified through the Neo RIP service that their bucket is not available as of yet.

*Testing done:*

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

#### General

- [X] I have read the [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md) doc
- [X] I certify that the changes I am introducing will be backward compatible, and I have discussed concerns about this, if any, with the Python SDK team
- [X] I used the commit message format described in [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md#committing-your-change)
- [X] I have passed the region in to all S3 and STS clients that I've initialized as part of this change.
- [X] I have updated any necessary documentation, including [READMEs](https://github.com/aws/sagemaker-python-sdk/blob/master/README.rst) and [API docs](https://github.com/aws/sagemaker-python-sdk/tree/master/doc) (if appropriate)

#### Tests

- [X] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [X] I have added unit and/or integration tests as appropriate to ensure backward compatibility of the changes
- [X] I have checked that my tests are not configured for a specific region or account (if appropriate)
- [X] I have used [`unique_name_from_base`](https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/utils.py#L77) to create resource names in integ tests (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by malav-shastri — APPROVED** [review]

**Review by tlian25 — APPROVED** [review]

File: src/sagemaker/jumpstart/constants.py [hunk 1]
```diff
@@ -70,6 +70,11 @@
             gated_content_bucket="jumpstart-private-cache-prod-eu-central-1",
             neo_content_bucket="sagemaker-sd-models-prod-eu-central-1",
         ),
+        JumpStartLaunchedRegionInfo(
+            region_name="eu-central-2",
+            content_bucket="jumpstart-cache-prod-eu-central-2",
+            gated_content_bucket="jumpstart-private-cache-prod-eu-central-2",
+        ),
```
Comment (AWS-pratab) [inline src/sagemaker/jumpstart/constants.py:77]: Is the neo bucket not needed for this region?

Comment (ErnevSharma) [reply src/sagemaker/jumpstart/constants.py:77]: Added to description: Adding JS bucket info. I verified through the Neo RIP service that their bucket is not available as of yet.

File: src/sagemaker/image_uri_config/djl-neuronx.json [hunk 2]
```diff
@@ -169,6 +176,7 @@
                 "cn-north-1": "727897471807",
                 "cn-northwest-1": "727897471807",
                 "eu-central-1": "763104351884",
+                "eu-central-2": "380420809688",
```
Comment (malav-shastri) [inline src/sagemaker/image_uri_config/djl-neuronx.json:179]: are there any E2E tests covering this file? we should probably trigger those before merging.

**Review by malav-shastri — APPROVED** [review]

File: src/sagemaker/jumpstart/constants.py (hunk 1 above)
Comment (JGuinegagne) [reply src/sagemaker/jumpstart/constants.py:77]: Then should you remove the neo bucket?

**Review by nargokul — APPROVED** [review]

**Pull request merged**